packages/cli/lib/lib/entry.js

```diff
@@ -13,7 +13,7 @@ if (process.env.NODE_ENV === 'development') {
 		hotLoader.preact(interopDefault(require('preact')));
 	}
 	// only add a debug sw if webpack service worker is not requested.
-	if (!process.env.ADD_SW && 'serviceWorker' in navigator) {
+	if (process.env.ADD_SW === undefined && 'serviceWorker' in navigator) {
```

Comment:
Does this now mean specifying the option at all will disable it?

Reply:
yes that was for https://github.com/preactjs/preact-cli/issues/809

Reply:
Yeah but I don't think that would be very intuitive, maybe add a check for `ADD_SW === 'false'`?

Reply:
if we check for ADD_SW === 'false', then sw-debug will get added even with --no-sw flag passed in the run command

Reply:
Keep the current check as well 🙃

Reply:
what should we do if condition ADD_SW === 'false' is true?

Reply:
```suggestion
	if ((!process.env.ADD_SW || process.env.ADD_SW === 'false') && 'serviceWorker' in navigator) {
```

myb

Reply:
the intent was that if someone passes --no-sw (process.env.ADD_SW is false or 'false') then sw-debug.js is not registered since user is intentionally disabling it and most likely understands the benefits of service worker. Our use case is that we are using workbox so would like to not add sw-debug.js when developing locally.

Reply:
@ForsakenHarmony please let me know if you have more q